account_asset_analytic/__init__.py

```diff
@@ -0,0 +1,6 @@
+# -*- coding: utf-8 -*-
+# © 2016 Antonio Espinosa - <antonio.espinosa@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+# flake8: noqa
```

Comment:
This is because Flake8 (not in Travis but yes in other instances) raise a Warning. This clause is only used in ```__init__.py``` files